src/evict/evict_page.c

```diff
@@ -86,8 +88,12 @@ __wt_page_release_evict(WT_SESSION_IMPL *session, WT_REF *ref)
 			 */
 			WT_STAT_CONN_INCR(
 			    session, cache_eviction_force_delete);
```

Comment:
Looking at how this falls out, I think it makes sense to add time tracking for ``cache_eviction_force`` and ``cache_eviction_force_delete`` as well. Maybe it could be structured as:

```
/* Describe why using non-standard error handling */
ret = __wt_evict(session, ref, false);
__wt_epoch(session, &stop);
if (ret == 0) {
    if (too_big)
        ....
} else {
    ....
}
```

That's a bit of an odd ordering with the handling of error returns, so I think it would deserve a comment.